streams-activitypub-graph/src/main/scala/org/apache/streams/activitypub/graph/impl/WebfingerGraphImpl.scala

```diff
@@ -0,0 +1,92 @@
+package org.apache.streams.activitypub.graph.impl
+
+import org.apache.http.client.utils.URIBuilder
+import org.apache.jena.query.ParameterizedSparqlString
+import org.apache.jena.rdf.model.Model
+import org.apache.jena.riot.system.PrefixMapStd
+import org.apache.jena.sparql.exec.http.QueryExecutionHTTP
+import org.apache.jena.sparql.exec.http.QueryExecutionHTTPBuilder
+import org.apache.streams.activitypub.api.WebfingerApi
+import org.apache.streams.activitypub.api.pojo.WebfingerQueryRequest
+import org.apache.streams.activitypub.api.pojo.WebfingerQueryResponse
+import org.apache.streams.activitypub.graph.config.WebfingerGraphImplConfig
+import org.apache.streams.activitypub.util.AcctPrefixResourceToResourceURISwap
+import org.apache.streams.config.ComponentConfigurator
+
+import java.net.URI
+import scala.io.Source;
+
+object WebfingerGraphImpl {
+
+  private final val configurator: ComponentConfigurator[WebfingerGraphImplConfig] = new ComponentConfigurator(classOf[WebfingerGraphImplConfig])
+  final val config: WebfingerGraphImplConfig = configurator.detectConfiguration()
+  final val DEFAULT: WebfingerGraphImpl = new WebfingerGraphImpl(config)
+
```

Comment:
always have a singleton implementation but will be possible to instantiate additional with different configs in the same jvm